di_website/templates/general/general_page.html

```diff
@@ -0,0 +1,42 @@
+{% extends 'layouts/default.html' %}
+
+{% load static wagtailcore_tags wagtailimages_tags string_utils card_tags %}
+
+{% block hero %}
+    {% include 'includes/heroes/hero.html' %}
+{% endblock hero %}
+
+{% block body %}
+    <section class="section">
+        <div class="row row--narrow">
+            <div class="l-sidebar">
+                
+                {% include 'includes/partials/side_nav.html'%}
+
+                <div class="l-sidebar__body">
+                    <div class="is-typeset wysiwyg">
+                        {% for block in page.body %}
+                            {% include_block block %}
+                        {% endfor %}
+                    </div>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    {% include './includes/team_member_quote.html'%}
```

Comment:
Ah, I was looking at code in the earlier files without knowing where it was going to be applied.
Anyway, I think all of these blocks can be added generically under one `SectionStreamBlock` ... allows for flexible ordering of the page content, including the banners and duos.